diff --git a/src/lua/lito.toml b/src/lua/lito.toml
index b5abeae..a209161 100644
--- a/src/lua/lito.toml
+++ b/src/lua/lito.toml
@@ -4,8 +4,7 @@ version = "5.5.1"
 standard = "c99"
 
 [external-sources.lua]
-archive = "https://www.lua.org/ftp/lua-5.5.1.tar.gz"
-sha256 = "1c4b4068d67061f2a2231ad2b5422e77acea1487ea9890f6320af614f4373dce"
+path = "LUA_PATH"
 
 [source-groups.lua-runtime]
 external-source = "lua"
